composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Razorpay Magento 2.0 plugin for accepting payments.",
     "version": "4.0.0",
     "require": {
-        "php": "~5.5.0|~5.6.0|7.*|^8.0",
+        "php": "~5.5.0|~5.6.0|7.*|^8.1",
```

Comment:
@enishant, are we still supporting magento 2.x for php 5.x? Can you please check which version of Magento still using php 5.x

Reply:
@ramth05 7.1, 7.2, 7.3, 7.4, 8.1 are supported php versions.
https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements.html
https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html